docs/ldml/tr35-general.md

```diff
@@ -900,6 +900,16 @@ The formal syntax for identifiers is provided below.
         <li>See Section 6.6 <a href="#Private_Use_Units">Private-Use Units</a></li>
     </ul></td></tr>
 
+<tr><td>number_prefix</td><td>:=</td>
+    <td>("1"[0-9]+ | [2-9][0-9]*) "-"
```

Comment:
I think we can accept `"1"` as a number prefix and assume it is exist by default if there is no number prefix.


Example: "1-meter-per-1-second"  equals "meter-per-second"
this will make no need to check if the number prefix is only one or not and it will be more easier in the coding.